packages/polindrome/presentation/lib/state_man/data/home_data.dart

```diff
@@ -0,0 +1,7 @@
+class HomeData {
+   bool? isPalindromeOrNot;
```

Comment:
I think it would be better to change the name of this variable to 'isPalindrome'